main/asciidoc/APKBUILD

```diff
@@ -21,24 +25,20 @@ build() {
 		--prefix=/usr \
 		--sysconfdir=/etc \
 		--mandir=/usr/share/man \
-		--infodir=/usr/share/info \
-		|| return 1
-	make || return 1
+		--infodir=/usr/share/info
+	make
 }
 
 package() {
 	cd "$builddir"
-	make DESTDIR="$pkgdir" install || return 1
+	make DESTDIR="$pkgdir" install
 }
 
 vim() {
-	depends=""
```

Comment:
Just spotted this. Resetting depends is required. Please don't remove this. After this it looks good to go for me.

Reply:
Fixed